src/config.rs

```diff
@@ -134,6 +134,11 @@ pub struct DbOptions {
     /// Configuration options for the compactor.
     pub compactor_options: Option<CompactorOptions>,
     pub compression_codec: Option<CompressionCodec>,
+
+    /// Configuration options for the disk cache. If set, when reading from object
+    /// storage, SlateDB will cache the data on disk. This can reduce the number of
+    /// network calls to object storage and speed up reads.
+    pub disk_cache_root_folder: Option<std::path::PathBuf>,
```

Comment:
Shouldn't we also take max size that can be used on the cache folder?

Reply:
That's covered in this issue:

https://github.com/slatedb/slatedb/issues/159